consensus/beacon/consensus.go

```diff
@@ -213,7 +214,7 @@ func (beacon *Beacon) VerifyUncles(chain consensus.ChainReader, block *types.Blo
 // 	   - nonce is expected to be 0
 //     - unclehash is expected to be Hash(emptyHeader)
 //     to be the desired constants
-// (b) the timestamp is not verified anymore
+// (b) we don't verify if a block is in the future anymore
```

Comment:
Perhaps we can say that:

(b) Ensure the block's timestamp is greater than its parent, but timestamp validity is not checked here